lib/listeners/statful.js

```diff
@@ -29,7 +19,7 @@ function dispatchMetricLine(listener) {
             });
 
             if (listener.config.stats) {
-                listener.statfulClient.counter('metrics_flushed', countMetricsInLines(lines));
+                listener.statfulClient.counter('metrics_flushed', linesList.length);
```

Comment:
But the number of metrics flushed equals the number of lines in the payload string.. What's the difference between counting lines and counting "\n" and add 1?